index.html

```diff
@@ -1,20 +1,268 @@
+      integrity="sha384-q2kxQ16AaE6UbzuKqyBE9/u/KzioAlnx2maXQHiDX9d4/zp8Ok3f+M7DPm+Ib6IU"
+      crossorigin="anonymous"
+    ></script>
+    <!-- Link for Boostrap JS -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.min.js"
+      integrity="sha384-pQQkAEnwaBkjpqZ8RU1fF1AKtTcHJwFl3pblpTlHXybJjHpMYo79HY3hIi4NKxyj"
+      crossorigin="anonymous"
+    ></script>
+    <!-- Font Awesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <!-- Google Fonts Exo 2 -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Exo+2&display=swap" rel="stylesheet">
+    <!-- Link to CSS file -->
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Bootstrap Trial</title>
+  </head>
 
-    <title>Responsive grid project</title>
-</head>
-
-<body>
-
-    <!-- Add your markup here -->
-
-</body>
+  <body>
+    <header>
+      <!-- Navigation bar  -->
+      <nav
+        class="navbar navbar-expand-lg navbar-light bg-white align-items-start"
+      >
+        <div class="container d-inline-flex">
+          <div class="row">
+            <div
+              class="col-xs-12 col-sm-8 col-md-6 col-lg-4 text-nowrap text-center"
+            >
+              <a class="navbar-brand" href="#"
+                ><span style="font-size: 20px; color: rgb(107, 10, 136)"
+                  >&#9639;</span
+                >
+                <b>Laura Menendez</b></a
+              >
+              <p>TRAINEE DEVELOPER</p>
+            </div>
+          </div>
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarNavAltMarkup"
+            aria-controls="navbarNavAltMarkup"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div
+            class="collapse navbar-collapse justify-content-center text-center"
+            id="navbarNavAltMarkup"
+          >
+            <div class="navbar-nav">
+              <a class="nav-link active" aria-current="page" href="index.html"
+                >About</a
+              >
+              <a class="nav-link" href="resume.html">Resume</a>
+              <a class="nav-link" href="portfolio.html">Portfolio</a>
+              <a class="nav-link" href="contact.html">Contact</a>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
 
+    <main>
+      <div class="container mt-5">
+        <div class="row">
+          <div class="col-md-offset-4 col-md-5">
+            <!-- Business card-->
+            <div class="container business-card shadow p-sm-3">
+              <div class="row">
```

Comment:
I can see that you are using a lot of divs with class name: row, which are most of the time not needed as you already have a div tag surrounding a section of the design